core/templates/services/questions-list.service.ts

```diff
@@ -17,141 +17,130 @@
  * questions list in editors.
  */
 
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+import cloneDeep from 'lodash/cloneDeep';
+
+import { AppConstants } from 'app.constants';
 import { EventEmitter } from '@angular/core';
+import { FormatRtePreviewPipe } from 'filters/format-rte-preview.pipe';
+import { QuestionBackendApiService } from
+  'domain/question/question-backend-api.service';
+import { QuestionSummaryForOneSkill, QuestionSummaryForOneSkillObjectFactory } from
+  'domain/question/QuestionSummaryForOneSkillObjectFactory';
+import { TruncatePipe } from 'filters/string-utility-filters/truncate.pipe';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class QuestionsListService {
+  private _questionSummariesForOneSkill: QuestionSummaryForOneSkill[] = [];
+  private _nextCursorForQuestions: string = '';
+  private _currentPage: number = 0;
+  private _questionSummartiesInitializedEventEmitter: EventEmitter<void> = (
+    new EventEmitter<void>());
+
+  constructor(
+    private formatRtePreviewPipe: FormatRtePreviewPipe,
+    private questionBackendApiService: QuestionBackendApiService,
+    private questionSummaryForOneSkillObjectFactory:
+    QuestionSummaryForOneSkillObjectFactory,
```

Comment:
Indent by 2 more spaces.
```suggestion
      QuestionSummaryForOneSkillObjectFactory,
```

Reply:
done